### ISSUE TITLE
Attribute selection

### ISSUE THREAD

**Issue by shichongrui** [uniqname/postpone#1]
There seems to be an issue with line 18 of your coffee script:

```
attr = el.hasAttribute('data-postpone') ? 'data-postpone' : 'postpone';
```

It is getting compiled into this mess:

```
attr = (_ref = el.hasAttribute('data-postpone')) != null ? _ref : {
    'data-postpone': 'postpone'
};
```

Which causes attr to be true if data-postpone is there and the object if it's not which is definitely not the intention.

It would seem as though CoffeeScript does not support ternary operators.
http://stackoverflow.com/questions/10146080/ternary-operation-in-coffeescript

**Comment by uniqname** [comment]
Ha! Interesting. I haven't touched this in ages, but it seems I had that fixed locally but never push it up. Thanks!

**Comment by uniqname** [comment]
fixed in [247746d1aa](https://github.com/uniqname/postpone/commit/247746d1aa8452bdb50080605142e50cae6b1133)
